libcxx/test/support/test_macros.h

```diff
@@ -7,6 +7,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+// This header contains several configuration defines.
```

Comment:
Leftover?

Reply:
No this should be a separate patch, which I thought this was. Removing it here.